src/js/components/UnitHealthNodesTable.js

```diff
@@ -82,23 +83,13 @@ class UnitHealthNodesTable extends React.Component {
   }
 
   getNodeLink(node, linkText) {
-    let parentPaths = this.props.parentRouter.getCurrentRoutes();
-    let parentPath = parentPaths[parentPaths.length - 2].name;
-    let path = 'system-overview-units-unit-nodes-node-panel';
-    // Route is available under dashboard root
-    if (parentPath === 'dashboard') {
-      path = 'dashboard-units-unit-nodes-node-panel';
-    }
-
-    let params = {unitID: this.props.itemID, unitNodeID: node.get('host_ip')};
+    let params = Object.assign(this.props.params, {unitNodeID: node.get('host_ip')});
```

Comment:
let's have an empty object as the first arg to `Object.assign`. we're modifying `this.props.params` here. don't know if that'll have consequences